File: Assignment3/Customer.java

/**
 * Elizabeth Gress T00738901
 * COMP 1231 Assignment 3
 * 01/31/2024
 */
public class Customer
{
    private int age;
    private double rating;

    /* Constructor for objects of class Customer */
    public Customer()
    {
        /* initialize instance variables */
        age = 0;
        rating = 0.0;
    }
    
    /* Overloaded constructor */
    public Customer(int age, double rating) 
    {
        this.age = age;
        this.rating = rating;

    }
    
    public int getAge()
    {
        return age;
    }
    
    public double getRating()
    {
        return rating;
    }
}

File: Programming Methods/C#/Chapter10/PreferredCustomer/Customer.cs
/* 
Elizabeth House
T00738901
COMP 2211 Programming Methods
C# Assignment 5
08/10/2024
*/

namespace Assignment5
{
	public class Customer : Person
	{
		/* Private variables for customer-specific information */
		private string _customerNum;
		private bool _mailList;

		/* Property to get or set the customer's address */
		public string CustomerNum
		{
			get { return _customerNum; }
			set { _customerNum = value; }
		}

		/* Property to get or set the customer's mailing list preference */
		public bool MailList
		{
			get { return _mailList; }
			set { _mailList = value; }
		}

		/* Default constructor initializes fields */
		public Customer()
		{
			_customerNum = string.Empty;
			_mailList = false;
		}
	}
}
